src/k8s-job-exit-spec/config/k8s-job-exit-spec.yaml

```diff
@@ -917,6 +917,20 @@ spec:
   - "Wait result from next retry"
   - "Decrease per task ports request"
 
+- code: 254
+  phrase: ContainerImageNotExists
+  issuer: PAI_RUNTIME
+  causer: PAI_RUNTIME
+  type: USER_FAILURE
+  stage: LAUNCHING
+  behavior: PERMANENT
+  reaction: NEVER_RETRY
+  reason: "User requests image is not exists"
+  repro:
+  - "Submit job with fake image uri"
```

Comment:
Maybe "incorrect Docker image uri" is better?

Reply:
fixed